### PR TITLE
New version: LinwoodCloud.Butterfly.Nightly version 2.1.1-rc.0

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/153507)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   0A7F0756FE0BAB41CF8301E95002D678AFBFBCBE382F15DA62BBF05861B71D56
 Expected hash: cc03a1e8b41be17f9741ba20d7a47848534cd355a1bc1ab7863126a9a5455936
 
 5/14/2024 9:15:47 AM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 881.)

File: manifests/l/LinwoodCloud/Butterfly/Nightly/2.1.1-rc.0/LinwoodCloud.Butterfly.Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ReleaseDate: 2024-05-14
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/LinwoodDev/Butterfly/releases/download/v2.1.1-rc.0/linwood-butterfly-windows-setup-x86_64.exe
+  InstallerSha256: CC03A1E8B41BE17F9741BA20D7A47848534CD355A1BC1AB7863126A9A5455936
```
Comment (stephengillie) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/2.1.1-rc.0/LinwoodCloud.Butterfly.Nightly.installer.yaml:19]:
```suggestion

```

(Automated response - build 881.)

File: manifests/l/LinwoodCloud/Butterfly/Nightly/2.1.1-rc.0/LinwoodCloud.Butterfly.Nightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/LinwoodDev/Butterfly/releases/download/v2.1.1-rc.0/linwood-butterfly-windows-setup-x86_64.exe
+  InstallerSha256: CC03A1E8B41BE17F9741BA20D7A47848534CD355A1BC1AB7863126A9A5455936
```
Comment (stephengillie) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/2.1.1-rc.0/LinwoodCloud.Butterfly.Nightly.installer.yaml:25]:
```suggestion

```

(Automated response - build 881.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-05-14 13:11:34.602 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.33135.0

(Automated response - build 881.)

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/149642
The issue is caused by the dependency not having a `user` scope.

**Comment by Trenly** [comment]
* #152555

**Pull request closed without merging**